### PR TITLE
feat(frontend): add CSV export for the subscription list

### PR DESCRIPTION
**Pull request by fkm-adfinis**
This list is needed by backoffice and sales and is currently
manually copy/pasted into their tool of choice.

### CODE REVIEW

File: frontend/app/ui/subscriptions/list/controller.js [hunk 1]
```diff
@@ -49,6 +52,39 @@ export default class SubscriptionsListController extends Controller {
...
+      this.intl.t("page.subscriptions.list.table.customer"),
+      this.intl.t("page.subscriptions.list.table.billing"),
+      this.intl.t("page.subscriptions.list.table.time-purchased"),
+      this.intl.t("page.subscriptions.list.table.time-spent"),
+      this.intl.t("page.subscriptions.list.table.time-total"),
+      this.intl.t("page.subscriptions.list.table.time-unconfirmed"),
+    ].join(",");
+
+    const lines = this.projects
+      .toArray()
```
Comment (velrest) [inline frontend/app/ui/subscriptions/list/controller.js:67]: Is the `toArray` required here?

File: frontend/app/ui/subscriptions/list/template.hbs [hunk 2]
```diff
@@ -92,5 +92,19 @@
       </table>
     </div>
 
+    <p class="subscription-export">
+      <button
+        {{on "click" this.export}}
+         class="subscription-export__button"
```
Comment (velrest) [inline frontend/app/ui/subscriptions/list/template.hbs:98]: Indentation is off here

Comment (velrest) [inline frontend/app/ui/subscriptions/list/template.hbs:98]: Indentation is off here

**Review by velrest — CHANGES_REQUESTED** [review]

File: frontend/app/ui/subscriptions/list/controller.js (hunk 1 above)
Comment (fkm-adfinis) [reply frontend/app/ui/subscriptions/list/controller.js:67]: Yes. I don't remember what exactly was wrong but did I try without before.

File: frontend/app/ui/subscriptions/list/template.hbs (hunk 2 above)
Comment (fkm-adfinis) [reply frontend/app/ui/subscriptions/list/template.hbs:98]: Interesting! `block-indentation` seems to have missed that. I'll correct it.

Comment (fkm-adfinis) [reply frontend/app/ui/subscriptions/list/template.hbs]: Done

**Review by velrest — APPROVED** [review]

**Pull request merged**